packages/editor/src/components/widgets/table/cell/TableCell.tsx

```diff
@@ -1,4 +1,8 @@
 import './TableCell.css';
 import type { ReactNode } from 'react';
 
-export const TableCell = (props: { children: ReactNode }) => <td className='table-cell'>{props.children}</td>;
+export const TableCell = (props: { children: ReactNode; cellWidth?: number }) => (
+  <td className='table-cell' style={props.cellWidth ? { width: `${props.cellWidth}px` } : {}}>
+    {props.children}
+  </td>
+);
```

Comment:
I would export all td properties and set this with on the TableCell directly...

```ts
export const TableCell = ({children, props}: React.HTMLAttributes<HTMLTableCellElement>) => (
  <td {...props} className='table-cell'>
    {children}
  </td>
);
```

```ts
<TableCell key={cell.id} style={{width: cell.column.getSize()}}>
  {flexRender(cell.column.columnDef.cell, cell.getContext())}
</TableCell>
```

Reply:
when i do it like you suggest i get the error: Property 'props' does not exist on type 'HTMLAttributes<HTMLTableCellElement>'

isn't it maybe like this?
`export const TableCell = ({ children, ...props }: React.HTMLAttributes<HTMLTableCellElement>) => (
  <td {...props} className='table-cell'>
    {children}
  </td>
);`

Reply:
yes, didn't wrote the code in an ide only here. so probably wrong. Only the idea ;)